zipline/utils/input_validation.py

```diff
@@ -25,6 +25,43 @@
 from zipline.utils.preprocess import preprocess
 
 
+def verify_indices_all_unique(obj):
```

Comment:
I've wanted this logic elsewhere before, so it felt reasonable to put it in the `input_validation` utils.